assets/kube-state-metrics/deployment.yaml

```diff
@@ -65,7 +65,7 @@ spec:
         - --upstream=http://127.0.0.1:8081/
         - --tls-cert-file=/etc/tls/private/tls.crt
         - --tls-private-key-file=/etc/tls/private/tls.key
-        image: quay.io/brancz/kube-rbac-proxy:v0.10.0
+        image: quay.io/brancz/kube-rbac-proxy:v0.9.0
```

Comment:
It is downgraded as there is a bug in the upstream VERSION file. Soon there will be 0.10.1 and this issue will be fixed.